bentoml/cli/__init__.py

```diff
@@ -144,6 +145,23 @@ def deploy(archive_path, platform, region, stage):
                 'Deploying with "--platform={platform}" is not supported in the current version of BentoML'
                 .format(platform=platform))
 
+    # Example useage: bentoml delete-deployment SERVICE_NAME --platform=aws-lambda
+    @_cli.command()
+    @click.argument('archive-path', type=click.STRING)
+    @click.option('--platform', type=click.Choice([
+        'aws-lambda', 'aws-lambda-py2', 'gcp-function', 'aws-sagemaker', 'azure-ml', 'algorithmia'
+    ]), required=True)
+    @click.option('--region', type=click.STRING, required=True)
+    @click.option('--stage', type=click.STRING)
+    def delete_deployment(archive_path, platform, region, stage):
+        if platform in SERVERLESS_PLATFORMS:
+            delete_serverless_deployment(platform, archive_path, region, stage)  # pylint:disable=too-many-function-args
```

Comment:
Recommend to check the deployment status given (platform, archive_path, region, stage) before calling delete, if there is no existing deployment found it should stop here and let the user know.

Reply:
done